### PR TITLE
User Administration

### PR DESCRIPTION
**Pull request by ingalls**
### Context

Add a UI modal to allow an administrator to modify user accounts

- Change username
- Change email
- Change access level
- disable account

Only mergable after https://github.com/mapbox/Hecate/pull/210 lands

cc/ @ingalls

### CODE REVIEW

**Comment by ingalls** [comment]
Screenshot from 2019-11-16 15-16-53
Screenshot from 2019-11-16 15-16-49

**Comment by ingalls** [comment]
- Add `Insufficient Permission` dialogue to modal for Users Listing & Webhook listing, avoiding throwing an error message for 401 error
- Standardize error handling & status code additions to error object in Main.js API

Screenshot from 2019-11-17 13-04-50

File: src/schema.sql [hunk 1]
```diff
@@ -39,7 +39,7 @@ CREATE INDEX bounds_idx ON bounds(name);
 DROP TABLE IF EXISTS users;
 CREATE TABLE users (
     id          BIGSERIAL PRIMARY KEY,
-    access      TEXT,
+    access      TEXT NOT NULL,
```
Comment (miccolis) [inline src/schema.sql:42]: pls no database changes if they can be avoided.

File: web/index.html [hunk 2]
```diff
@@ -21,10 +21,6 @@
     </head>
     <body class='h-full'>
         <div id="app" class="h-full relative">
-            <div class='w240 align-center'>
-                LOGIN
-
-            </div>
```
Comment (miccolis) [inline web/index.html:27]: Can I get some context on this deletion? Is it just dead code?

**Review by miccolis — CHANGES_REQUESTED** [review]
As per chat looks like there is a missing db migration.

File: src/schema.sql (hunk 1 above)
Comment (miccolis) [reply src/schema.sql:42]: Ignore... is needed.

File: web/index.html (hunk 2 above)
Comment (ingalls) [reply web/index.html:27]: Yup, this was code I added when I was testing out the full auth mode that we currently have deployed. VueJS takes control of the parent div and wipes anything inside of it (name this). So you would briefly see `LOGIN` flash by on the screen if you're browser was a mite slow.

**Review by miccolis — APPROVED** [review]
Everything looks good to me. However, please confirm that this migration has been tested on staging!

**Comment by ingalls** [comment]
Tested on staging and everything seems to be in order!

**Pull request merged**